package.json

```diff
@@ -34,10 +34,14 @@
   },
   "version": "1.0.0",
   "devDependencies": {
-    "coveralls": "^3.0.0",
     "chai": "^4.1.2",
-    "jscs": "^3.0.7",
-    "jshint": "^2.9.5",
+    "coveralls": "^3.0.0",
+    "eslint": "4.18.1",
```

Comment:
Can these please be updated to use the `^` notation? That way it's implied that we expect our project to work with all future `patch` and `minor` releases.

Reply:
I will do that. Forgot to undo some recent npm configuration.